pallets/xcm-helper/src/lib.rs

```diff
@@ -76,10 +88,39 @@ pub mod pallet {
     #[pallet::pallet]
     pub struct Pallet<T>(_);
 
+    #[pallet::event]
+    #[pallet::generate_deposit(pub(super) fn deposit_event)]
+    pub enum Event<T: Config> {
+        /// Sent staking.bond call to relaychain
+        Bonding(T::AccountId, BalanceOf<T>, RewardDestination<T::AccountId>),
+        /// Sent staking.bond_extra call to relaychain
+        BondingExtra(BalanceOf<T>),
+        /// Sent staking.unbond call to relaychain
+        Unbonding(BalanceOf<T>),
+        /// Sent staking.rebond call to relaychain
+        Rebonding(BalanceOf<T>),
+        /// Sent staking.withdraw_unbonded call to relaychain
+        WithdrawingUnbonded(u32),
+        /// Sent staking.nominate call to relaychain
+        Nominating(Vec<T::AccountId>),
+    }
+
     #[pallet::error]
     pub enum Error<T> {
         /// Xcm message send failure
         SendXcmError,
+        /// Failed to send staking.bond call
+        BondFailed,
+        /// Failed to send staking.bond_extra call
+        BondExtraFailed,
```

Comment:
we can remove these errors now and keep only `SendXcmError`

Reply:
Can we do it as a a part of #1053 ?

Reply:
sure